.github/workflows/deploy_dev.yml

```diff
@@ -16,5 +16,4 @@ jobs:
       external_role_arn: ${{ secrets.external_role_arn }}
       vector_secret_name: ${{ secrets.vector_secret_name }}
       vector_vpc_id: ${{ secrets.vector_vpc_id }}
-      vector_security_group: ${{ secrets.vector_security_group }}
-
+      vector_security_group: ${{ secrets.vector_security_group }}
```

Comment:
new line

Reply:
can you also add these changes to `deloy_staging.yml` ?

Reply:
although we maynot have values for vector staging yet

Reply:
yeah that's why I didn't include it just yet. But could include it for completeness